osquery/tables/system/linux/os_version.cpp

```diff
@@ -22,7 +22,13 @@ namespace xp = boost::xpressive;
 namespace osquery {
 namespace tables {
 
-#if defined(REDHAT_BASED)
+#if defined(RHEL)
+const std::string kLinuxOSRelease = "/etc/redhat-release";
+const std::string kLinuxOSRegex =
+    "(?P<name>[\\w+\\s]+) .* "
+    "(?P<major>[0-9]+)\\.(?P<minor>[0-9]+) "
+    "(?P<patch>\\(\\w+\\))";
+#elif defined(CENTOS)
```

Comment:
change this to `#elif defined(REDHAT_BASED)` to also include the Oracles, Amazon Linuxi. Are you positive this patch only applies to RHEL and is not needed on CentOS?